DataConnectors/Imperva WAF Gateway/Connector_Imperva_WAF_Gateway.json

```diff
@@ -0,0 +1,119 @@
+{
+  "id": "ImpervaWAFGateway",
+  "title": "Imperva WAF Gateway",
+  "publisher": "Imperva",
+  "descriptionMarkdown": "The [Imperva](https://www.imperva.com) connector will allow you to quickly connect your Imperva WAF Gateway alerts to Azure Sentinel. This provides you additional insight into your organization's WAF traffic and improves your security operation capabilities.",
+  "graphQueries": [
+      {
+          "metricName": "Total data received",
+          "legend": "WAF_Alerts",
```

Comment:
Expected: Imperva WAF Gateway , DATATYPE_NAME. Please refer https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_CEF_template.json.

Reply:
@v-jayakal, please resolve the comment if it is fixed and you have confirmed the fix